docs/includes/register-sso-add-in-aad-v2-include.md

```diff
@@ -1,64 +1,126 @@
-## Create an app registration
+## Register the add-in with Microsoft identity platform
+
+You need to create an app registration in Azure that represents your web server. This enables authentication support so that proper access tokens can be issued to the client code in JavaScript. This registration supports both SSO in the client, and fallback authentication using the Microsoft Authentication Library (MSAL).
 
-Registering your application (the add-in) establishes a trust relationship between your add-in and the Microsoft identity platform. The trust is unidirectional: your add-in trusts the Microsoft identity platform, and not the other way around.
 
 1. Sign in to the [Azure portal](https://portal.azure.com/) with the ***admin*** credentials to your Microsoft 365 tenancy. For example, **MyName@contoso.onmicrosoft.com**.
-1. Under **Manage**, select **App registrations** > **New registration**. On the **Register an application** page, set the values as follows.
+1. Select the **App registrations** icon. If you don't see the icon, search for "app registration" in the search bar.
```

Comment:
```suggestion
1. Select **App registrations**. If you don't see the icon, search for "app registration" in the search bar.
```
Per MSFT style, refer to UI elements by their label and not the element. Unless it's totally confusing not to do so.
https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33421